bokehjs/src/coffee/models/plots/gmap_plot_canvas.coffee

```diff
@@ -110,8 +110,13 @@ export class GMapPlotCanvasView extends PlotCanvasView
     # create the map with above options in div
     @map = new maps.Map(@canvas_view.map_el, map_options)
 
-    # update bokeh ranges whenever the map idles, which should be after any UI action
-    maps.event.addListenerOnce(@map, 'idle', @_set_bokeh_ranges)
+    # update bokeh ranges whenever the map idles, which should be after most UI action
+    maps.event.addListener(@map, 'idle',
+      @_set_bokeh_ranges
+      @render())
```

Comment:
@bryevdv, ^^^ this doesn't make sense. I presume this should be:
```
maps.event.addListener @map, 'idle', () =>
  @_set_bokeh_ranges()
  @render()
```
Obviously the original compiles as it's perfectly valid CoffeeScript. I hate this language.